src/dune_rules/package.ml

```diff
@@ -101,28 +106,29 @@ module Dependency = struct
   module Constraint = struct
     module Var = struct
       type t =
-        | QVar of string
+        | Literal of string
         | Var of string
 
       let encode = function
-        | QVar v -> Dune_lang.Encoder.string v
+        | Literal v -> Dune_lang.Encoder.string v
```

Comment:
If possible, could you extract the rename into a separate PR?

Reply:
Yep! https://github.com/ocaml/dune/pull/7733